mixins/fetch-pennsieve-file/index.js

```diff
@@ -12,7 +12,27 @@ export default {
       const filesResponse = await axios.$get(filesUrl)
       const files = filesResponse.files
       filePath = filePath.toLowerCase()
-      return files.find(element => filePath === element.path.toLowerCase() || filePath.includes(element.uri.substring(element.uri.lastIndexOf('/')).toLowerCase()))
+      let foundFile = {}
+
+      files.forEach(file => {
+        // Check if path matches query param
+        if (filePath.toLowerCase() === file.path.toLowerCase) {
+          foundFile = file
+        }
```

Comment:
If you find a file here you are still going to carry on with further checks.  Is this what you intended.

Reply:
I agree, I think this should be an if/else-if

Reply:
Thanks guys you are correct. Fixed now!